build-tools/tasks/styles.js

```diff
@@ -74,7 +74,6 @@ function stylesTask(theme) {
       variablesMap,
       scssDir: workspace.sourcePath,
       componentsOutputDir: theme.outputPath,
-      skip: designTokensOutputDir ? [] : ['design-tokens'],
```

Comment:
`designTokensOutputDir` is always a truthy value (see above where it is defined), so this condition is never false and not needed at all